README.md

```diff
@@ -94,7 +91,11 @@ Note that if you get errors, run the above commands with `--verbose` - this will
 
 ## Development
 
-This modules has been created for educational reasons, so any updates to the documentation are welcome.
+In order to aid in development a java based emulator is included within the `spec/fictures` folder, allowing you to more easily test and develop the code within this module. In order to start it up simply enter the following into the command line:
+```
+java -jar ./spec/fixtures/HueEmulator-v0.8.jar
+```
+Once it is running and before pressing start be sure to change the port number to `80`, as this is the port that the module is configured to work with.
```

Comment:
This comment could be updated with the recent port changes :-)

Reply:
updated